Remotion/Data/DomainObjects.UnitTests/Database/DataDomainObjects_TestDomain_SetupDB.sql

```diff
@@ -1159,3 +1159,106 @@ CREATE VIEW [dbo].[SupplierView] ([ID], [ClassID], [Timestamp], [Name], [Industr
 GO
 -- Create indexes for tables that were created above
 -- Create synonyms for tables that were created above
+-- Create all structured types
+IF TYPE_ID('[dbo].[TVP_String]') IS NULL CREATE TYPE [dbo].[TVP_String] AS TABLE
```

Comment:
There is a problem in the test database setup: several `RdbmsProviderDefinitions` use the same connection string. Since the TableTypes are created for each `RdbmsProviderDefinition`, this leads to conflicts.
(We avoid this issue for entities, because each entity belongs to a single `RdbmsProviderDefinition`.)